skywalking/__init__.py

```diff
@@ -40,6 +40,7 @@ class Component(Enum):
     Sanic = 7007
     AioHttp = 7008
     Pyramid = 7009
+    Falcon = 7010
```

Comment:
This must be an old original source commit since the Component IDs 7010 and 7011 have been reserved for a bit:
```
    Psycopg = 7010
    Celery = 7011
```